Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/createChannel.md

$createChannel[name;type;(optional) categoryID]
```

### Parameters

> - `name` `(Type : String || Flag : Required)` : The name to give this channel.

Reviewer: ```suggestion
- `name` `(Type: String || Flag: Required)`: The name of the new channel.
```

    > 📝 Channel names can have a maximum of upto 100 characters.

> - `type` `(Type : Enum || Flag : Required)` : The channel type. Channel types:

Reviewer: ```suggestion
- `type` `(Type: Enum || Flag: Required)`: The channel type. Channel types:
```
